src/app/users/users.component.ts

```diff
@@ -21,6 +21,12 @@ import { debug } from '../debug-operator';
     .mat-column-select {

```

Comment:
We'd like to set the column max-width similar to this `.mat-column-select` class but for `.mat-column-profile`.  We should revert the img tag to use the `.profile-image` class.